float64[3] init_com_pos
float64[3] init_com_orient
float64[3] final_com_pos
float64[3] final_com_orient
float64[3] init_lankle_pos
float64[3] init_lankle_orient
float64[3] final_lankle_pos
float64[3] final_lankle_orient
float64[3] init_rankle_pos
float64[3] init_rankle_orient
float64[3] final_rankle_pos
float64[3] final_rankle_orient
float64 time
float64 dt
---
int32 duration